cmd/flarectl/.goreleaser.yml

```diff
@@ -0,0 +1,33 @@
+before:
+  hooks:
+    - go mod download
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - arm
+      - arm64
+archives:
+  - replacements:
+      darwin: Darwin
+      linux: Linux
+      windows: Windows
```

Comment:
What do we get by capitialising the values here? If it's related to the filename, it's probably best to leave it lower case and avoid any case sensitivity dramas.

Reply:
It's generated by default by Goreleaser (I don't really know why).